### PR TITLE
affinityitems version 2.0.0

### PR DESCRIPTION
**Pull request by mathieulgb**

### CODE REVIEW

File: upgrade/Upgrade-2.0.0.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+if (!defined('_PS_VERSION_'))
+	exit;
+
+function upgrade_module_2_0_0($module)
+{
+	Configuration::updateValue('AE_BULK_PACKAGE', '50');
+	Configuration::updateValue('AE_FUNNEL', 1);
+	Configuration::updateValue('AE_TRACKING_JS', 0);
+	Configuration::updateValue('AE_LIMIT', 2);
+	return true;
```
Comment (Quetzacoalt91) [inline upgrade/Upgrade-2.0.0.php]: You updated the database content in the file *configuration/sqlinstall.php*. These "INSERT" actions will only be applied on the module installation, so you should insert the same lines if the merchant update its module.

File: upgrade/Upgrade-2.0.0.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	}
```
Comment (Quetzacoalt91) [inline upgrade/Upgrade-2.0.0.php]: Don't you want to execute these SQL requests after that ? :)

**Comment by Quetzacoalt91** [comment]
Hi @mathieulgb,

When you update your PR, don't forget to add a short message or I will never be notified by your progress.

**Comment by mathieulgb** [comment]
i just complete the upgrade script and added image preview on simple activation

**Comment by Quetzacoalt91** [comment]
Hi @mathieulgb,

Thanks, the PR has just been merged and the changes should be available from tomorrow.

Best regards

**Pull request merged**